editor/gui/editor_scene_tabs.cpp

```diff
@@ -135,6 +135,16 @@ void EditorSceneTabs::_scene_tab_input(const Ref<InputEvent> &p_input) {
 	}
 }
 
+void EditorSceneTabs::unhandled_key_input(const Ref<InputEvent> &p_event) {
+	if (tab_preview_panel->is_visible()) {
```

Comment:
Can be early return:
```suggestion
	if (!tab_preview_panel->is_visible()) {
		return;
	}
```

Reply:
Fixed.